### PR TITLE
Refactor/endpoints

### PR DESCRIPTION
**Pull request by raop155**
# Description

- Refactor: `API` routes

### CODE REVIEW

File: apps/web/src/app/api/accounts/[address]/route.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const { address } = params || {};
+
+    return NextResponse.json(await getWallet(address, session?.user?.email));
+  } catch (error) {
+    logger.error(error);
+    return NextResponse.error();
+  }
+}
+
+export async function PATCH(request: NextRequest, { params }: { params: { address: string } }) {
```
Comment (elranu) [inline apps/web/src/app/api/accounts/[address]/route.ts:24]: this should be PUT I think

File: apps/web/src/shared/utils/EIP155RequestHandlerUtil.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export async function approveEIP155Request(requestEvent: RequestEventArgs) {
     case EIP155_SIGNING_METHODS.ETH_SIGN:
       try {
         const message = getSignParamsMessage(request.params);
-        const signedMessageResponse = await fetch(`/wallet/${addr}/personalSign`, {
+        const signedMessageResponse = await fetch(`api//wallet/${addr}/personal-sign`, {
```
Comment (elranu) [inline apps/web/src/shared/utils/EIP155RequestHandlerUtil.ts]: two slash here: "api//wallet"

File: apps/web/src/shared/utils/EIP155RequestHandlerUtil.ts [hunk 3]
```diff
@@ -58,7 +58,7 @@ export async function approveEIP155Request(requestEvent: RequestEventArgs) {
       try {
         const transaction = request.params[0];
         debugger;
-        const signedMessageResponse = await fetch(`/wallet/${addr}/ethSendTransaction`, {
+        const signedMessageResponse = await fetch(`api//wallet/${addr}/eth-send-transaction`, {
```
Comment (elranu) [inline apps/web/src/shared/utils/EIP155RequestHandlerUtil.ts]: same here

Comment (raop155) [reply apps/web/src/shared/utils/EIP155RequestHandlerUtil.ts]: @elranu Fixed

**Review by elranu — APPROVED** [review]

**Pull request merged**